hw-1/task1.js

```diff
@@ -0,0 +1,19 @@
+import * as readline from 'readline';
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+rl.on('line', (text) => {
+  console.log(`${text} -> ${reverse(text)}`);
+});
+
+
+/**
```

Comment:
Comments are mostly redundant in node js - you just need to name your methods to represent what they are doing. But it's ok, you can keep adding comments if you want

Reply:
Gotcha. Removed it.